### PR TITLE
build: Setup nightly run of cache fuzzer

### PR DESCRIPTION
**Pull request by kKPulla**
Differential Revision: D67356608

### CODE REVIEW

File: .github/workflows/scheduled.yml [hunk 1]
```diff
@@ -313,6 +313,13 @@ jobs:
           path: velox/_build/debug/velox/functions/prestosql/fuzzer/velox_window_fuzzer_test
           retention-days: "${{ env.RETENTION }}"
 
+      - name: Upload cache fuzzer
+        uses: actions/upload-artifact@v4
+        with:
+          name: cache
```
Comment (kgpai) [inline .github/workflows/scheduled.yml:319]: Give it a more relevant name , like cache-fuzzer-artifact

File: .github/workflows/scheduled.yml [hunk 2]
```diff
@@ -706,6 +713,42 @@ jobs:
...
+    steps:
+
+      - name: Download cache fuzzer
+        uses: actions/download-artifact@v4
+        with:
+          name: cache
+
+      - name: Run cache Fuzzer
+        run: |
+          cat /proc/sys/vm/max_map_count
```
Comment (kgpai) [inline .github/workflows/scheduled.yml:732]: nit why the cat ?

**Review by kgpai — APPROVED** [review]
Minor nits: looks good.

File: .github/workflows/scheduled.yml (hunk 1 above)
Comment (kKPulla) [reply .github/workflows/scheduled.yml:319]: I was following the naming style like others in the file. Ex: window, join, aggregation etc. Do you recommend changing this still, may be call it "cache-fuzzer"?

File: .github/workflows/scheduled.yml (hunk 2 above)
Comment (kKPulla) [reply .github/workflows/scheduled.yml:732]: removed

**Pull request closed without merging**